charts/alfresco-transform-service/templates/config-libreoffice.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 data:
   {{- include "alfresco-transform-service.sfs.url" . | indent 2 }}
   {{- toYaml .Values.libreoffice.environment | nindent 2 }}
-  {{- range $key, $val := .Values.libreoffice.livenessProbe }}
+  {{- range $key, $val := omit .Values.libreoffice.livenessProbe "path" }}
```

Comment:
Why did we remove it from router but not from sfs?

Reply:
good catch, but please write comments on the appropriate line :D

Reply:
oops :p